pkg/api/types.go

```diff
@@ -121,3 +121,26 @@ type InstallResult struct {
 	// Error describes last error encountered during install operation
 	Error error
 }
+
+// SourceInfo stores information about the source code
+type SourceInfo struct {
+	// Store the current GIT ref which might be a branch or commit id
+	// This information is available in the image as 'io.openshift.build.commit.ref' LABEL
+	Ref string
+
+	// The GIT commit id
+	// This information is available in the image as 'io.openshift.build.commit.id' LABEL
+	ID string
```

Comment:
Pls rename this to to `CommitID`, since `ID` is too vague IMHO. I assume the actual output data will match the name of this field, right?